quinn-proto/src/connection.rs

```diff
@@ -742,7 +742,6 @@ impl Connection {
         &mut self,
         mux: &mut impl Multiplexer,
         now: u64,
-        remote: SocketAddrV6,
```

Comment:
I don't mind removing this for now, but I should note that we'll need it back eventually to support migration.

Reply:
Yeah, I just want it to be warning-clean...